packages/utils/src/schema/getDefaultFormState.ts

```diff
@@ -194,12 +213,22 @@ export function computeDefaults<
             get(defaults, [key]),
             rootSchema,
             get(formData, [key]),
-            includeUndefinedValues === "excludeObjectChildren"
+            passAsFalseForIncludeUndefinedValues.includes(
+              includeUndefinedValues
+            )
               ? false
               : includeUndefinedValues
```

Comment:
Instead of inlining this can we set it before L210?
```ts
          const nestedIncludeUndefinedValues = passAsFalseForIncludeUndefinedValues.includes(
              includeUndefinedValues
            )
              ? false
              : includeUndefinedValues
          // Compute the defaults for this node, with the parent defaults we might
          // have from a previous run: defaults[key].
          const computedDefault = computeDefaults<T, S, F>(
            validator,
            get(schema, [PROPERTIES_KEY, key]),
            get(defaults, [key]),
            rootSchema,
            get(formData, [key]),
            nestedIncludeUndefinedValues
          );
```

Also, couldn't
```ts
passAsFalseForIncludeUndefinedValues.includes(includeUndefinedValues) ? false : includeUndefinedValues
```
be simplified to this?
```ts
includeUndefinedValues === true
```

Reply:
Yes, yes it could. Great simplification